contrib/TraceLogHelper/JsonParser.cs

```diff
@@ -80,8 +80,9 @@ private static Event ParseEvent(XElement xEvent, string file, bool keepOriginalE
 				TraceFile = file,
 				DDetails = xEvent.Elements()
 					.Where(a=>a.Name != "Type" && a.Name != "Time" && a.Name != "Machine" && a.Name != "ID" && a.Name != "Severity" && (!rolledEvent || a.Name != "OriginalTime"))
-					.ToDictionary(a=>string.Intern(a.Name.LocalName), a=>(object)a.Value),
-				original = keepOriginalElement ? xEvent : null,
+					// When the key contains a colon character, it gets parsed as a:item
+					.ToDictionary(a=>a.Name.LocalName == "item" ? a.Attribute("item").Value : string.Intern(a.Name.LocalName), a=>(object)a.Value),

```

Comment:
Can you describe what this is doing? I'm trying to understand what effect it may have on other tooling we have that uses these parsers.

Reply:
This is really a hack.
In order to minimize the changes to the existing code,
it converts everything read from JSON into XML internally.
When it converts a JSON KV with the key containing a colon character, it recognizes the colon character as a namespace specifier, and creates an XML element "item" with the original name in the attribute value.
This is actually my "best guess" as I couldn't find a documentation describing this behavior.

Reply:
Ah, or maybe the colon is simply not a valid character?  Either way, it just creates this anonymous class "item".

Reply:
Oh I think I see what you mean, thanks.